api/src/users/controllers/users-controller.ts

```diff
@@ -65,9 +65,13 @@ class UsersController {
         return this.usersService.findOneByIdAndUserJwtId(id, userId);
     }
 
-    @ApiUnauthorizedResponse()
-    @ApiForbiddenResponse()
     @ApiOkResponse()
+    @ApiUnauthorizedResponse({ description: "Invalid JWT token" })
+    @ApiForbiddenResponse({ description: "You are not the owner of account" })
+    @ApiInternalServerErrorResponse({
+        description:
+            "If something goes wrong with deleting file on server, but it's very little chance"
```

Comment:
`it's very` -> `there is`

Reply:
Error, can occure but it's probably impossible, 99% to not occure.